appinventor/blocklyeditor/src/msg/en/_messages.js

```diff
@@ -1304,6 +1304,7 @@ Blockly.Msg.en.switch_language_to_english = {
     Blockly.Msg.REPL_NOT_NOW = "Not Now";
     Blockly.Msg.REPL_COMPANION_OUT_OF_DATE1 = "The Companion you are using is out of date.<br/><br/>This Version of App Inventor should be used with Companion version";
     Blockly.Msg.REPL_COMPANION_OUT_OF_DATE_IMMEDIATE = "You are using an out-of-date Companion. You should update the MIT AI2 Companion as soon as possible. If you have auto-update setup in the store, the update will happen by itself shortly.";
+    Blockly.Msg.REPL_COMPANION_WRONG_PACKAGE = "Make sure you are using the correct MIT AI2 Companion";
```

Comment:
Probably better to say
The Companion you are using was built for different instance of App Inventor.  To obtain the correct companion look on the App Inventor screen under Help and Companion Information.

-- 
And ace we make sure the companion will always be there?